src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Definition/RazorDefinitionEndpoint.cs

```diff
@@ -282,7 +282,7 @@ private async Task<Range> GetNavigateRangeAsync(DocumentSnapshot documentSnapsho
             }
 
             var attributeDescriptor = (propertyName is not null)
-                ? originTagDescriptor.BoundAttributes.FirstOrDefault(a => a.Name.Equals(propertyName, StringComparison.Ordinal))
+                ? originTagDescriptor.BoundAttributes.FirstOrDefault(a => a.Name?.Equals(propertyName, StringComparison.Ordinal) == true)
```

Comment:
This scenario can't really happen in practice because we only support go-to-definnition in Blazor / on components today BUT `IndexerNamePrefix`es (the only thing that can cause `Name` to be `null`) only apply to non-Blazor scenarios